.github/workflows/mobile-perf.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         CXX: /opt/llvm/bin/clang++
     steps:
     - uses: actions/checkout@v4
+    - name: Ensure files don't leak back into the main binary
+      run: rm source/extensions/listener_managers/listener_manager/listener_manager_impl.h source/server/overload_manager_impl.cc
```

Comment:
should this be in this PR?

Reply:
nevermind, I didn't read this thoroughly, I see this does belong in this PR, pls ignore